mods/ts/sequences/infantry.yaml

```diff
@@ -1,37 +1,37 @@
-e1:
+e1.gdi:
 	Defaults:
 		Tick: 80
-	stand:
+	stand: e1
```

Comment:
Why not specifying this -and other infantry filenames - on the Defaults instead of the current override on every subsequence?